### PR TITLE
Update fub_curta.config and fub_curta.md

### PR DESCRIPTION
**Pull request by wassimsalam01**
- Formatted fub_curta.config for consistency
- Fixed in fub_curta.md broken link when opening methylseq profile from https://nf-co.re/configs/fub_curta/

### CODE REVIEW

**Comment by jfy133** [comment]
Thanks for this @wassimsalam01 ! I'll merge this in after the  mega PR (as it's coming from my fork)

File: docs/fub_curta.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ In addition, job submissions are assigned the appropriate quality of service (QO
 
 Institute-specific pipeline profiles exists for:
 
-- [methylseq](pipeline/methylseq/fub_curta.md)
+- [methylseq](https://github.com/wassimsalam01/configs/blob/master/docs/pipeline/methylseq/fub_curta.md)
```
Comment (jfy133) [inline docs/fub_curta.md:29]: This should point to the main repo, not your fork - is there any reason why you're keeping the docs on your fork?

Comment (wassimsalam01) [reply docs/fub_curta.md]: Done!

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
Thanks @wassimsalam01 !

**Pull request merged**